fundz/config.py

```diff
@@ -1,10 +1,7 @@
-import os
-
-ENV = os.getenv('ENVIRONMENT', 'development')
-DEBUG = bool(os.getenv('DEBUG', True))
-
-SQLALCHEMY_DATABASE_URI = os.getenv(
-    'DATABASE_URI',
-    'postgres://postgres:postgres@localhost/fundz'
-)
-SQLALCHEMY_TRACK_MODIFICATIONS = False
+def map_config(config):
+    return {
+        'ENV': config['default']['ENVIRONMENT'],
```

Comment:
Do we need a default section, or would bare key/val pairs be okay? How would other sections work in this config model?

Reply:
Whelp, nevermind, `ConfigParser` won't parse it without headers. That's annoying.

Reply:
Yeah, unfortunately those seem to be required.